grid2op/utils/l2rpn_idf_2023_scores.py

```diff
@@ -103,32 +104,41 @@ def __init__(
             nb_process_stats=nb_process_stats,
             scores_func={
                 "grid_operational_cost": L2RPNSandBoxScore,
+                "assistant_confidence": _AlertTrustScore,
+                # "assistant_cost": _AlertCostScore,
                 "new_renewable_sources_usage": _NewRenewableSourcesUsageScore,
             },
             score_names=["grid_operational_cost_scores",
+                         "assistant_confidence_scores",
```

Comment:
shouldn't "confidence" be renamed to "trust"?

Reply:
Perhaps a good idea, keeping as it was set originally for now (probably Clement naming, should see with him)